packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -79,15 +103,24 @@ export default class FlyoutHelpStore {
 
         const getNextBlock = async (xml, current_index, direction) => {
             const next_index = current_index + (direction ? 1 : -1);
-            const block_type = Object.keys(xml).find((key, index) => next_index === index);
-
-            if (!block_type) {
+            const next_blocks_Type = Object.keys(xml).filter((key, index) =>
+                direction ? next_index <= index : next_index >= index
+            );
+            const next_filled_bock = await this.getFilledHelpContents(next_blocks_Type);
+
+            const next_filled_bock_lenght = next_filled_bock.length;
```

Comment:
- Do we need to put this into a variable?
- Typo

```suggestion
            const next_filled_bock_length = next_filled_bock.length;
```

Reply:
done

Reply:
done.